doc/mkdocs/docs/api/basic_json/default_object_comparator_t.md

```diff
@@ -0,0 +1,18 @@
+# <small>nlohmann::basic_json::</small>default_object_comparator_t
+
+```cpp
+using default_object_comparator_t = std::less<StringType>;  // until C++14
+
+using default_object_comparator_t = std::less<>;            // since C++14
+```
+
+The default comparator used by [`object_t`](object_t.md).
+
+Since C++14 a transparent comparator is used which prevents unnecessary string construction.
```

Comment:
```suggestion
Since C++14, a transparent comparator is used which prevents unnecessary string construction.
```

Reply:
Actually, that is grammatically correct. Since + verb and noun would be followed by a comma, since + noun phrase doesn't need one.

While comma placement is a definite weakness of mine, be it in my native language or English, this one I did get right. :-)

On a different technical note I just realized that we're never checking for the presence of `is_transparent` in our comparator. I'll check if and where that is needed.

Reply:
Thanks - it's before 9 am and I already learned something :-)

Reply:
The current implementation checks via `is_comparable` whether to SFINAE-enable various functions. This ignores possible conversions and temporaries. It would be better to add `is_transparent` as an additional requirement. It then forces the user to explicitly convert if desired.

Reply:
I guess that was part of all attempts so far... Let's hope you're more successful!

Reply:
> I guess that was part of all attempts so far... Let's hope you're more successful!

Tests succeed. Just need to finish the documentation and I'm procrastinating. ;-)

(Same goes for C++20 stuff … even more documentation. Yikes.)

Reply:
Thank you so much for your huge effort and your persistence!